doc/source/examples/extended_examples/gui/executable.rst

```diff
@@ -0,0 +1,111 @@
+.. _gui_example:
+
+=======================================
+Create a GUI app in Python with PySide6
+=======================================
+
+This example shows how to create a graphical user interface (GUI) app in Python that uses PyMAPDL to compute the deflection of a square beam.
+
+Simulation setup
+================
+
+The :download:`gui.py <gui.py>` script launches a graphical app using PySide6.
+The preprocessing tab contains input fields for Poisson's ratio, Young modulus, beam length, and number of simulation nodes.
+
+
+.. image:: base_app.png
+
+Add a PyVista plotting frame in the window
+==========================================
+
+Start by importing `QtInteractor <https://qtdocs.pyvista.org/api_reference.html#qtinteractor>`_ from ``pyvistaqt`` and :class:`MapdlTheme <ansys.mapdl.core.theme.MapdlTheme>` from ``ansys-mapdl-core``.
+
+.. code:: python
+
+    from pyvistaqt import QtInteractor
+    from ansys.mapdl.core import MapdlTheme
+
+Then, add a plotter in the first tab:
+
+.. code:: python
+
+    def _setup_tab_preprocessing(self) -> None:
+        ...
+        container_layout.addWidget(self._run_preprocessor_button, 5, 0, 1, 3)
+
+        # PyVista frame in the window
+        self._preprocessing_plotter = QtInteractor(theme=MapdlTheme())
+        container_layout.addWidget(self._preprocessing_plotter, 0, 4, 6, 50)
+
+And another one in the second tab:
+
+.. code:: python
+
+    def _setup_tab_postprocessing(self) -> None:
+        container_layout = QtWidgets.QVBoxLayout()
+        self._tab_postprocessing.setLayout(container_layout)
+        self._postprocessing_plotter = QtInteractor(theme=MapdlTheme())
+        container_layout.addWidget(self._postprocessing_plotter)
+        self._deflection_label = QLabel("Deflection: ")
+        container_layout.addWidget(self._deflection_label)
+
+Finally, make sure to correctly close the VTK widgets when closing the app:
+
+.. code:: python
+
+    def closeEvent(self, event) -> None:
+        self._preprocessing_plotter.close()
+        self._postprocessing_plotter.close()
+        event.accept()  # let the window close
+
+Launch an MAPDL instance in your window
+=======================================
+
+Add an attribute to your MainWindow for the MAPDL instance and import ``launch_mapdl``.
```

Comment:
```suggestion
Add an attribute to your MainWindow for the MAPDL instance and import the ``launch_mapdl`` package.
```
Change "MainWindow" to main window or tag it in backticks to make it a literal (and follow it with a noun) describing what it is.)